backend/benefit/applications/models.py

```diff
@@ -260,3 +281,71 @@ class Meta:
         db_table = "bf_applications_applicationbasis"
         verbose_name = _("application basis")
         verbose_name_plural = _("application bases")
+
+
+class Employee(UUIDModel, TimeStampedModel):
+    application = models.OneToOneField(
+        Application,
+        verbose_name=_("application"),
+        related_name="employee",
+        on_delete=models.CASCADE,
+    )
+
+    first_name = models.CharField(max_length=128, verbose_name=_("first name"))
+    last_name = models.CharField(max_length=128, verbose_name=_("last name"))
+    social_security_number = models.CharField(
```

Comment:
Kesäseteli uses an encrypted field for the SSN, should we do that too?

Reply:
Good point, this should be encrypted, fixed